src/Evolution/Systems/Burgers/BoundaryConditions/DirichletAnalytic.hpp

```diff
@@ -94,6 +105,59 @@ class DirichletAnalytic final : public BoundaryCondition {
     return {};
   }
 
+  using fd_interior_evolved_variables_tags = tmpl::list<>;
+  using fd_interior_temporary_tags =
+      tmpl::list<evolution::dg::subcell::Tags::Mesh<1>>;
+  using fd_gridless_tags = tmpl::list<
+      evolution::dg::subcell::Tags::Coordinates<1, Frame::ElementLogical>,
+      ::Tags::Time, domain::Tags::FunctionsOfTime,
+      domain::Tags::ElementMap<1, Frame::Grid>,
+      domain::CoordinateMaps::Tags::CoordinateMap<1, Frame::Grid,
+                                                  Frame::Inertial>,
+      fd::Tags::Reconstructor, ::Tags::AnalyticSolutionOrData>;
+
+  template <typename AnalyticSolutionOrData>
+  void fd_ghost(const gsl::not_null<Scalar<DataVector>*> u,
+                const Direction<1>& direction, const Mesh<1> subcell_mesh,
+                const tnsr::I<DataVector, 1, Frame::ElementLogical>&
+                    subcell_logical_coords,
+                const double time,
+                const std::unordered_map<
+                    std::string,
+                    std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+                    functions_of_time,
+                const ElementMap<1, Frame::Grid>& logical_to_grid_map,
+                const domain::CoordinateMapBase<Frame::Grid, Frame::Inertial,
+                                                1>& grid_to_inertial_map,
+                const fd::Reconstructor& reconstructor,
+                const AnalyticSolutionOrData& analytic_solution_or_data) const {
+    const size_t ghost_zone_size{reconstructor.ghost_zone_size()};
+
+    // slice logical coordinates and shift it to compute inertial coordinates of
+    // ghost points
+    auto shifted_logical_coords =
+        evolution::dg::subcell::slice_tensor_for_subcell(
+            subcell_logical_coords, subcell_mesh.extents(), ghost_zone_size,
+            direction);
+    const double delta_x{subcell_logical_coords.get(0)[1] -
```

Comment:
Here I used `get(0)` because of the data ordering. Otherwise for y or z coordinates grabbing `[1]` and `[0]` and subtracting those two would give zero. I think this would work at least now because we force subcell mesh to be isotropic in many places in code, but would be happy to generalize this.